src/BridgeMap.js

```diff
@@ -0,0 +1,36 @@
+const { REPRESENTATION } = require("./constants/values");
+const { STATE } = require("./constants/message");
+class BridgeMap {
+  #currentMap = { upperPart: [], lowerPart: [] };
+
+  updateMyPositionForward(direction, symbol) {
+    this.#fillPathAccordingInput(direction, symbol);
+    return this;
+  }
+
+  getMyBridgeMap() {
+    return this.#currentMap;
+  }
+
+  init() {
+    this.#currentMap = { upperPart: [], lowerPart: [] };
+  }
+
+  #fillPathAccordingInput(input, symbol) {
+    if (input === REPRESENTATION.UPPER.abbreviatedForm) {
+      this.#currentMap.upperPart.push(symbol);
+      this.#fillBlankUnselectedPath(REPRESENTATION.LOWER.numericalForm);
+    }
+    if (input === REPRESENTATION.LOWER.abbreviatedForm) {
+      this.#currentMap.lowerPart.push(symbol);
+      this.#fillBlankUnselectedPath(REPRESENTATION.UPPER.numericalForm);
+    }
```

Comment:
```suggestion
  if (input === REPRESENTATION.UPPER.abbreviatedForm) {
    this.#currentMap.upperPart.push(symbol);
    this.#fillBlankUnselectedPath(REPRESENTATION.LOWER.numericalForm);
    return;
  }

  this.#currentMap.lowerPart.push(symbol);
  this.#fillBlankUnselectedPath(REPRESENTATION.UPPER.numericalForm);
```
앞선 `if`문에서 `return`을 해준다면 뒤따르는 `if`문을 생략할 수 있을 것 같습니다..!